fluent-package/templates/usr/sbin/fluentd.erb

```diff
@@ -12,4 +12,52 @@ if ARGV.include?("--version")
   puts "fluent-package #{PACKAGE_VERSION} fluentd #{Fluent::VERSION} (#{FLUENTD_REVISION})"
   exit 0
 end
+
+class DuplicateInstanceChecker
+  def running_fluentd?
+    IO.popen(["systemctl", "is-active", "fluentd"]) do |_io|
+      return true if _io.readlines.last.chomp == "active"
+    end
+    false
+  end
+
+  def prevent_duplicate_launch?
+    # check whether main process or not.
+    # e.g. directly spawned from shell
+    unless running_fluentd?
+      return false
+    end
+    begin
+      ppid = Process.ppid
+      if IO.read("/proc/#{ppid}/cmdline").split("\u0000").include?("fluentd")
+        # worker
+	return false
+      else
+        return true
+      end
+    rescue
+      return true
+    end
+    false
+  end
```

Comment:
Thanks for this fix!
I'm concerned about this.
Does this mean `/usr/sbin/fluentd` cab be called from the supervisor process?
Can we reproduce this by just restarting the service?

Reply:
No. 
This issue is raised when `/usr/sbin/fluentd` was invoked.
(service file use /opt/fluent/bin/fluentd)

Reply:
Note: We don't need this check for workers, this check has been removed.